<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmCreateMDADataset</class>
 <widget class="QDialog" name="frmCreateMDADataset">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>765</width>
    <height>397</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>765</width>
    <height>397</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>시계열 다차원 데이터셋 생성</string>
  </property>
  <widget class="QTableWidget" name="tbSelItem">
   <property name="geometry">
    <rect>
     <x>504</x>
     <y>28</y>
     <width>257</width>
     <height>165</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::SingleSelection</enum>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectRows</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>62</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>* 지표 목록</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>204</y>
     <width>86</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>* 공간 범위</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tbItem">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>29</y>
     <width>405</width>
     <height>169</height>
    </rect>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::ExtendedSelection</enum>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectRows</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>508</x>
     <y>9</y>
     <width>62</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>* 선택 지표</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tbSid">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>233</y>
     <width>401</width>
     <height>160</height>
    </rect>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectRows</enum>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>92</y>
     <width>77</width>
     <height>54</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <widget class="QPushButton" name="btnAdd">
      <property name="text">
       <string>추가</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="btnDel">
      <property name="text">
       <string>제거</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>236</y>
     <width>341</width>
     <height>149</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>시간범위 :</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QComboBox" name="cboSyear"/>
    </item>
    <item row="0" column="3">
     <widget class="QComboBox" name="cboEyear"/>
    </item>
    <item row="0" column="2">
     <widget class="QLabel" name="label_4">
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="text">
       <string>~</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>공간단위 :</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2" colspan="2">
     <widget class="QPushButton" name="btnOk">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>50</height>
       </size>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="text">
       <string>생성</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1" colspan="3">
     <widget class="QComboBox" name="cboLevel"/>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>tbItem</tabstop>
  <tabstop>btnAdd</tabstop>
  <tabstop>btnDel</tabstop>
  <tabstop>tbSelItem</tabstop>
  <tabstop>tbSid</tabstop>
  <tabstop>cboSyear</tabstop>
  <tabstop>cboEyear</tabstop>
  <tabstop>cboLevel</tabstop>
  <tabstop>btnOk</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
